src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 20px;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
```

Comment:
You are applying `margin-left` to all `.nav_item` elements except the first one. This is a good approach, it's better to add margins to one side (either left or right) to keep the code cleaner and avoid any possible issues with margin collapsing.